[2023-11-20T08:38:36.291+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:38:36.298+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:38:36.302+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:38:36.324+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-06 00:00:00+00:00
[2023-11-20T08:38:36.330+0000] {standard_task_runner.py:57} INFO - Started process 77 to run task
[2023-11-20T08:38:36.341+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-06T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpxj6k20h4']
[2023-11-20T08:38:36.352+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask retrieve_data
[2023-11-20T08:38:36.419+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [running]> on host 74ca1234e792
[2023-11-20T08:38:36.527+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-06T00:00:00+00:00'
[2023-11-20T08:38:36.530+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Scripts/etl.py", line 18, in retrieve_data
    aws_access_key_id=YOUR_ACCESS_KEY,
NameError: name 'YOUR_ACCESS_KEY' is not defined
[2023-11-20T08:38:36.560+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231106T000000, start_date=20231120T083836, end_date=20231120T083836
[2023-11-20T08:38:36.574+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task retrieve_data (name 'YOUR_ACCESS_KEY' is not defined; 77)
[2023-11-20T08:38:36.596+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-20T08:38:36.632+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:43:23.136+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:43:23.149+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:43:23.149+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:43:23.160+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-06 00:00:00+00:00
[2023-11-20T08:43:23.168+0000] {standard_task_runner.py:57} INFO - Started process 81 to run task
[2023-11-20T08:43:23.174+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-06T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp5i4sdpm7']
[2023-11-20T08:43:23.186+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask retrieve_data
[2023-11-20T08:43:23.263+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [running]> on host c84b35a13244
[2023-11-20T08:43:23.365+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-06T00:00:00+00:00'
[2023-11-20T08:43:25.933+0000] {etl.py:60} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:43:25.935+0000] {etl.py:64} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:43:26.082+0000] {etl.py:71} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:43:26.491+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:43:26.506+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231106T000000, start_date=20231120T084323, end_date=20231120T084326
[2023-11-20T08:43:26.552+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:43:26.608+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:46:36.580+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:46:36.592+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:46:36.594+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:46:36.622+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-06 00:00:00+00:00
[2023-11-20T08:46:36.631+0000] {standard_task_runner.py:57} INFO - Started process 87 to run task
[2023-11-20T08:46:36.643+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-06T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpm872azx7']
[2023-11-20T08:46:36.651+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask retrieve_data
[2023-11-20T08:46:36.749+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [running]> on host d26cbef9e8ab
[2023-11-20T08:46:36.853+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-06T00:00:00+00:00'
[2023-11-20T08:46:39.298+0000] {etl.py:62} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:46:39.301+0000] {etl.py:66} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:46:39.493+0000] {etl.py:73} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:46:40.090+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:46:40.097+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231106T000000, start_date=20231120T084636, end_date=20231120T084640
[2023-11-20T08:46:40.120+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:46:40.151+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:50:16.709+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:50:16.718+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [queued]>
[2023-11-20T08:50:16.723+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:50:16.809+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-06 00:00:00+00:00
[2023-11-20T08:50:16.812+0000] {standard_task_runner.py:57} INFO - Started process 80 to run task
[2023-11-20T08:50:16.818+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-06T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpt3eh6hlj']
[2023-11-20T08:50:16.822+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask retrieve_data
[2023-11-20T08:50:16.934+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-06T00:00:00+00:00 [running]> on host 3ef39241d6e6
[2023-11-20T08:50:17.050+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-06T00:00:00+00:00'
[2023-11-20T08:50:18.971+0000] {etl.py:87} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:50:18.972+0000] {etl.py:91} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:50:19.122+0000] {etl.py:98} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:50:20.153+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:50:20.162+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231106T000000, start_date=20231120T085016, end_date=20231120T085020
[2023-11-20T08:50:20.229+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:50:20.277+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
